backend/entityservice/views/project.py

```diff
@@ -151,20 +151,26 @@ def project_binaryclks_post(project_id):
                 # connexion has already read the data before our handler is called!
                 # https://github.com/zalando/connexion/issues/592
                 # stream = get_stream()
-                stream = BytesIO(request.data)
-                expected_bytes = binary_format(size).size * count
+                stream = iterable_to_stream(BytesIO(request.data))
```

Comment:
I'm confused. Isn't `BytesIO` already a stream? https://docs.python.org/3/library/io.html#binary-i-o

Reply:
yes it is. I got confused by the [read](https://docs.python.org/3.7/library/io.html?highlight=bufferedreader#io.BufferedIOBase.read) function, it says that it reads **up** to 'size' bytes. Thus I thought it might be saver to wrap it in a BufferedReader, but I guess that's unnecessary.